contentcuration/contentcuration/utils/publish.py

```diff
@@ -125,7 +126,7 @@ def assign_license_to_contentcuration_nodes(channel, license):
     channel.main_tree.get_family().update(license_id=license.pk)
 
 
-def map_content_nodes(
+def map_content_nodes(  # noqa: C901
```

Comment:
Ah that's how to do it with a multi-line arguments list

Reply:
It took me a couple of tries...